Find_The_Second_Largest_Number.rb

```diff
@@ -0,0 +1,28 @@
+=begin
+Create a function that takes a list of numbers and returns the second largest number.
+
+Examples
+
+second_largest([10, 40, 30, 20, 50]) ➞ 40
+
+second_largest([25, 143, 89, 13, 105]) ➞ 105
+
+second_largest([54, 23, 11, 17, 10]) ➞ 23
+=end
+
+
+def second_largest(array)
+  first_largest = array.max

```

Comment:
when you do this, you have already gone through the array to find the largest number. 

try doing this in one loop manually keep tracking and then return it in the end.